app/timetopaytaxpayer/controllers/TaxpayerController.scala

```diff
@@ -36,49 +36,18 @@ class TaxpayerController @Inject() (
     cc:           ControllerComponents
 )(implicit executionContext: ExecutionContext, clock: Clock) extends BackendController(cc) {
 
-  def getTaxPayer(utr: SaUtr): Action[AnyContent] = Action.async { implicit request =>
-
-    //start features before for comprehension
+  def getSelfAssessmentsAndDebits(utr: SaUtr): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
Could you rename it to `getReturnsAndDebits` please? We'd have a more consistent naming.